examples/ssd/train_multi.py

```diff
@@ -0,0 +1,234 @@
+import argparse
+import copy
+import numpy as np
+
+import chainer
+from chainer.datasets import ConcatenatedDataset
+from chainer.datasets import TransformDataset
+from chainer.optimizer import WeightDecay
+from chainer import serializers
+from chainer import training
+from chainer.training import extensions
+from chainer.training import triggers
+
+from chainercv.datasets import voc_bbox_label_names
+from chainercv.datasets import VOCBboxDataset
+from chainercv.extensions import DetectionVOCEvaluator
+from chainercv.links.model.ssd import GradientScaling
+from chainercv.links.model.ssd import multibox_loss
+from chainercv.links import SSD300
+from chainercv.links import SSD512
+from chainercv import transforms
+
+from chainercv.links.model.ssd import random_crop_with_bbox_constraints
+from chainercv.links.model.ssd import random_distort
+from chainercv.links.model.ssd import resize_with_random_interpolation
+
+
+class MultiboxTrainChain(chainer.Chain):
+
+    def __init__(self, model, alpha=1, k=3):
+        super(MultiboxTrainChain, self).__init__()
+        with self.init_scope():
+            self.model = model
+        self.alpha = alpha
+        self.k = k
+
+    def __call__(self, imgs, gt_mb_locs, gt_mb_labels, n_positive):
+        mb_locs, mb_confs = self.model(imgs)
+        loc_loss, conf_loss = multibox_loss(
+            mb_locs, mb_confs, gt_mb_locs, gt_mb_labels, self.k)
+        loss = loc_loss * self.alpha + conf_loss
+
+        chainer.reporter.report(
+            {'loss': loss, 'loss/loc': loc_loss, 'loss/conf': conf_loss},
+            self)
+
+        # use the number of positives in all GPUs instead of that in this GPU
+        with chainer.cuda.get_device_from_array(gt_mb_labels):
+            if n_positive[0] > 0:
+                loss *= (gt_mb_labels > 0).sum() / n_positive[0]
+        return loss
+
+
+class Transform(object):
+
```

Comment:
If this class is completely same as the training script for single GPU, there should be a comment saying it.